newdle/api.py

```diff
@@ -356,6 +358,29 @@ def update_participant(args, code, participant_code):
     for key, value in args.items():
         setattr(participant, key, value)
     db.session.commit()
+    formatted_answers = []
+    for date, state in participant._answers.items():
+        formatted_answers.append(
+            "Date: {} ({})".format(
+                date.replace("T", " at "), state.replace("ifneedbe", "if needed")
+            )
+        )
+
+    if participant.newdle.notify:
+        notify_newdle_creator(
+            participant,
+            f'{participant.name} replied to {participant.newdle.title}',
+            'joined_email.txt',
+            'joined_email.html',
+            {
+                'participant': participant.name,
+                'title': participant.newdle.title,
+                'answers': formatted_answers,
+                'summary_link': url_for(
+                    'newdle', code=participant.newdle.code + '/summary', _external=True
```

Comment:
define a new `build_only` url rule in `create_app` and use that in the `url_for` call here instead of adding path segments to the `code`